src/statistics/distributions/gaussian.rs

```diff
@@ -351,15 +353,17 @@ mod tests_gaussian {
         let dist: Gaussian = Gaussian::default();
 
         // Values from WolframAlpha
-        assert_approx_equal!(dist.cdf(-4.0), 0.000_031_671_241_835_663_76, EPS);
-        assert_approx_equal!(dist.cdf(-3.0), 0.001_349_898_031_574_464_2, EPS);
-        assert_approx_equal!(dist.cdf(-2.0), 0.022_750_131_947_162_62, EPS);
-        assert_approx_equal!(dist.cdf(-1.0), 0.158_655_253_945_057_25, EPS);
+        assert_approx_equal!(dist.cdf(-5.0), 2.866_515_718_791_939e-7, EPS);
```

Comment:
Why have these test values changed ? I pulled them from Wolfram Alpha.

Reply:
Hi, 
the test was not passing and the algorithm seemed to be correct. So I rechecked the values with
- `Wolfram Alpha` using `CDF[NormalDistribution[0, 1], x] to 17 digits` where x is the parameter
- `mpmath` using `mp.ncdf` and `mp.dps=20`

Both resulted in the same numbers, but the number was different from the previous ones.
I think that is the result of changing
```rust
       // from
        0.5 * erf::erfc(-(x - self.mean) / (SQRT_2 * self.variance.sqrt()));
       // to
        0.5 * (-(x - self.mean) / (SQRT_2 * self.variance.sqrt())).erfc();
        
```